lib/msal-browser/src/utils/MsalFrameStatsUtils.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { InProgressPerformanceEvent } from "@azure/msal-common/browser";
+
+export function collectInstanceStats(
+    currentClientId: string,
+    performanceEvent: InProgressPerformanceEvent
+): void {
+    const frameInstances: string[] =
+        // @ts-ignore
+        window.msal?.clientIds || [];
+
+    const msalInstanceCount = frameInstances.length;
+
+    let sameClientIdInstanceCount = 0;
+
+    for (const i of frameInstances) {
+        if (i === currentClientId) sameClientIdInstanceCount++;
+    }
```

Comment:
nit: A little more concise
```suggestion
    const sameClientIdInstanceCount = frameInstances.filter(i => i === currentClientId).length;
```